integreat_cms/cms/templates/users/region_user_form.html

```diff
@@ -91,11 +91,13 @@ <h3 class="heading font-bold text-black"><i data-feather="key" class="mr-2"></i>
                     {% render_field user_form.role|add_error_class:"border-red-500" required="required" %}
                     <div class="help-text">{{ user_form.role.help_text }}</div>
                 </div>
-                <div>
-                    <label for="{{ user_form.organization.id_for_label }}">{{ user_form.organization.label }}</label>
-                        {% render_field user_form.organization|add_error_class:"border-red-500" %}
-                    <div class="help-text">{{ user_form.organization.help_text }}</div>
-                </div>
+                {% if user_form.organization.field.queryset.exists %}
```

Comment:
Yes, but only Integreat staff members can add organizations - and there is currently no intention to create organizations since we wait until we have region-based organizations (see #1566).
I guess this just makes sense because it's the same mechanism as we applied for the page form:
https://github.com/digitalfabrik/integreat-cms/blob/9117e089de865a766e9f98f8634b1c06f6d6fbf0/integreat_cms/cms/templates/pages/page_form.html#L264-L274

And we can also keep this in the future, because the field does not make sense when there are no organizations.

Reply:
Okay, I agree on that. However, I would therefore change the permission to create system-wide organizations as this consequences might be unclear to our current service team members.